src/common/StatementMetadata.cpp

```diff
@@ -167,7 +167,7 @@ const char* StatementMetadata::getPlan(bool detailed)
 
 	if (plan->isEmpty())
 	{
-		UCHAR info[] = {detailed ? isc_info_sql_explain_plan : isc_info_sql_get_plan};
```

Comment:
Please no.
I understand that you solve some problem (not sure error or warning) with integer to unsigned char conversion.
IMO it should have better solution than placing explicit casts (does not matter C or C++ style) here and there.
If you can't instruct compiler to accept existing style please wait a bit - your solution works but it is not good from sources readability POV